pom.xml

```diff
@@ -1,17 +1,146 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>net.conjur.api</groupId>
+  <groupId>com.cyberark.conjur.api</groupId>
   <artifactId>conjur-api</artifactId>
-  <version>2.2.1</version>
+  <!-- Only bump version number,
+  SNAPSHOT is automatically removed during release deployments -->
+  <version>3.0.0-SNAPSHOT</version>
```

Comment:
does this mean we're bumping the version in this PR? we should be prepared to add a tag immediately after merge

will this be published in Maven as version `3.0.0-SNAPSHOT`? why not just version `3.0.0`?

Reply:
So, to clarify a bit what I was saying above, there are a few reasons this is important.

Firstly, when we kick off the publishing process with our plugins, it checks if the version includes 'SNAPSHOT' or not. If it does, it uploads to the SNAPSHOT repository in OSSRH. In addition to simplifying our pipeline, this acts as a safeguard against accidentally publishing a release.

Second, for `pom.xml` files, you want to include the `SNAPSHOT` suffix when it's anything other than a release or when building locally. Since our releases will only be consumed through Maven, we can assume that anyone who downloads our source code to compile locally will be using a SNAPSHOT tag. 

Finally, our release deployment script temporarily overwrites the version in the pom.xml for us, so we don't have to worry about removing then re-adding the SNAPSHOT tag.